src/mainviews/create-job.tsx

```diff
@@ -211,6 +245,8 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
             onChange={handleInputChange}
             value={textInputs['inputFile'] ?? props.model.inputFile}
             id={`${formPrefix}inputFile`}
+            onBlur={(e) => validateNonBlank(e.target)}
+            error={hasError('inputFile')}
```

Comment:
I believe you can pass the `helpText` in as well to display the error message itself.

Reply:
Yes, you're right!

![image](https://user-images.githubusercontent.com/93281816/192073107-9087d101-c7fe-45f4-bc15-76e38cd39d81.png)